Source/santad/SNTExecutionController.mm

```diff
@@ -404,8 +404,27 @@ - (void)validateExecEvent:(const Message &)esMsg postAction:(bool (^)(SNTAction)
           self->_ttyWriter->Write(targetProc, msg);
         }
 
+        void (^replyBlock)(BOOL) = ^void(BOOL authenticated) {
+        };
+
+        // Only allow a user in standalone mode to override a block if an
+        // explicit block rule is not set when using a sync service.
+        if (config.enableStandaloneMode &&
+            (!config.syncBaseURL || se.decision == SNTEventStateBlockUnknown)) {
```

Comment:
I'm not sure this is the condition we want - "standalone enabled and no sync server configured" would be enough to succeed the condition without checking the decision, potentially allowing for a standalone rule to be created. But what about the case of static rules? We'd go thru the flow of letting the user approve some binary, but it would still fail to run due to static rules having higher precedence which would likely cause confusion.

Reply:
If standalone is enabled and you do not have a sync service then local rule configuration is enabled via the command line via `sudo santactl rule`.

In which case the ability to bypass lockdown is tied to do you have root access (or at least sudo), admittedly this isn't always the same as having enrolled in Touch ID, so I probably need to add a check in the authorization part to ensure the user is in the `admin` group. 

If we want to prevent bypassing static rules we need a way to identify a rule as a static rule or make the check explicitly tied to `SNTEventStateBlockUnknown`.

Reply:
> We'd go thru the flow of letting the user approve some binary, but it would still fail to run due to static rules having higher precedence which would likely cause confusion.

Ah I misread this. That'd be super frustrating, to avoid this I think we need to know that the rule is from the static rules and cannot be modified. I'll open a separate PR for this as it's useful to have in the `santactl fileinfo`  output.